src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private final Random random = new Random();
+    private final int index = random.nextInt(Colors.values().length);
```

Comment:
The `index` variable should be a local variable inside the `getRandomColor` method rather than a class-level field. This is because each call to `getRandomColor` should produce a potentially different index to get a random color each time.